tasks/rh-sign-image/rh-sign-image.yaml

```diff
@@ -72,21 +72,14 @@ spec:
 
             referenceContainerImage=$(jq -r ".components[${COMPONENTS_INDEX}].containerImage" ${SNAPSHOT_PATH})
 
-            reference=$(jq -r ".components[${COMPONENTS_INDEX}].repository" ${SNAPSHOT_PATH})
+            # Use the rh-registry-repo directly from the snapshot
```

Comment:
I would probably remove this comment. Again, this sound more like it's describing the change, because previously it was done differently, but when somebody reads this code in the future, they won't know that it used to be any different.

Reply:
It was fixed, and the comment can be resolved. Thanks.